code/modules/admin/topic.dm

```diff
@@ -1648,4 +1648,74 @@ Status: [status ? status : "Unknown"] | Damage: [health ? health : "None"]
 			L.sleeping = 9999999
 
 		log_admin("[key_name(usr)] has [L.sleeping ? "enabled" : "disabled"] sleeping on [key_name(L)].")
-		message_admins("[ADMIN_TPMONTY(usr)] has [L.sleeping ? "enabled" : "disabled"] sleeping on [ADMIN_TPMONTY(L)].")
\ No newline at end of file
+		message_admins("[ADMIN_TPMONTY(usr)] has [L.sleeping ? "enabled" : "disabled"] sleeping on [ADMIN_TPMONTY(L)].")
+
+
+	else if(href_list["offer"])
+		if(!check_rights(R_ADMIN))
+			return
+
+		var/mob/living/L = locate(href_list["offer"]) in GLOB.mob_living_list
+
+		if(!istype(L))
```

Comment:
Same here, we'd probably want this logged.

Reply:
No

Reply:
Could you expand on the negative?

Reply:
No